app/javascript/packs/billboard.js

```diff
@@ -23,6 +23,11 @@ async function generateDisplayAd(element) {
     element.innerHTML = '';
     element.appendChild(generatedElement);
     setupDisplayAdDropdown();
+    // This is called here because the ad is loaded asynchronously.
+    // The original code is still in the asset pipeline, so is not importable.
```

Comment:
n/b: I don't know what the "original" code in this context is, and I wonder if elaborating on this comment somewhat would be useful? maybe this is just newbie-itis